pywwt/core.py

```diff
@@ -191,16 +191,36 @@ def _validate_longitude(self, proposal):
     ss_orbit_color = ColorWithOpacity('darkgray', help='The color of the solar system orbit paths (:class:`str` or `tuple`)').tag(wwt='solarSystemOrbitColor')
     ss_overlays = Bool(False, help='Whether to show solar system overlays (:class: `bool`)').tag(wwt='solarSystemOverlays') # what does this mean?
     ss_scale = Int(1, help='Specifies how to scale the size of the Sun and its planets, with 1 as actual size (:class: `int`)').tag(wwt='solarSystemScale')
-    ss_stars = Bool(False, help='Whether to show solar system overlays (:class: `bool`)').tag(wwt='solarSystemOverlays')
+    ss_stars = Bool(False, help='Whether to show solar system overlays (:class: `bool`)').tag(wwt='solarSystemOverlays')'''
 
+    def set_view(self, mode):
+        """
+        Change the view mode. (...)
+
+        Parameters
+        ----------
+        mode : `str`
+            The desired mode. (default: 'sky')
+        """
+        available = ['sky', 'sun', 'mercury', 'venus', 'earth', 'moon', 'mars',
+                     'jupiter', 'saturn', 'uranus', 'neptune', 'pluto']
+        ss_mode = '3D Solar System View'
+        
+        if mode in available:
+            self._send_msg(event='set_viewer_mode', mode=mode)
+        elif mode == 'solar_system':
```

Comment:
This is a personal issue with WWT, but the "3D solar system mode" has an infinite zoom, as well as SDSS Cosmos data. I think this might be a chance to call this mode '3D' or '3d universe' or something you all think is better than 'solar_system'

Reply:
OK, I'm open to suggestions on the name.